Monika After Story/game/script-topics.rpy

```diff
@@ -8314,3 +8314,47 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_poemherself",category=['ddlc','misc'],prompt="Poems for Monika",random=True))
+
+label monika_poemherself:
+    m 4ekc "Do you remember how there wasn't a chance to write a poem for my interests [player]?"
+    m 1rsc "I don't get it. Why did they ignored me?"
```

Comment:
Add "did" after "Why".
remove the "d" in "ignored"

Or

Add "have" after "Why", but the first one is likely better.